specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -327,6 +378,11 @@
           "description": "Resource ID of the database to store job metadata in.",
           "type": "string",
           "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {}
+            ]
+          },
```

Comment:
I'm not sure these lines about allowedResources empty add anything, do we need it?

Reply:
This was also added by engine